recipe/meta.yaml

```diff
@@ -30,17 +35,44 @@ test:
     - "\"%R%\" -e \"library('rTensor')\""  # [win]
 
 about:
-  home: http://jamesyili.github.io/rTensor
+  home: https://github.com/rikenbit/rTensor
   license: GPL-3.0-or-later
-  summary: A set of tools for creation, manipulation, and modeling of tensors with arbitrary number of modes. A tensor in the context of data analysis is a multidimensional array. rTensor does this by providing a S4 class 'Tensor' that wraps around the base 'array' class. rTensor provides common tensor operations as methods,
-    including matrix unfolding, summing/averaging across modes, calculating the Frobenius norm, and taking the inner product between two tensors. Familiar array operations are overloaded, such as index subsetting via '[' and element-wise operations. rTensor also implements various tensor decomposition, including CP, GLRAM,
-    MPCA, PVD, and Tucker. For tensors with 3 modes, rTensor also implements transpose, t-product, and t-SVD, as defined in Kilmer et al. (2013). Some auxiliary functions include the Khatri-Rao product, Kronecker product, and the Hamadard product for a list of matrices.
-  license_family: GPL
-  license_file: {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

Comment:
I'm not sure what to do here. The recipe originally bundled GPL-3. But now the cran skeleton wants to bundle GPL-2. Do we have strong feelings on how to interpret the CRAN license designation "GPL (>=2)"? I've seen conflicting examples, which makes me suspect that the behavior of the CRAN skeleton may have changed over time.

Note: The SPDX designation of GPL-3.0-or-later was added later, and was presumably chosen based on the fact that `license_file` indicated GPL-3, even though the CRAN metadata would indicate GPL-2.0-or-later.

Reply:
The CRAN skeleton changed over time and was not always optimal in the past:

CRAN meta-data `# License: GPL (>= 2)` clearly evaluates to spdx: `license: GPL-2.0-or-later` (to be changed above).

As `>= 2` does only specify a lower bound version, the only stable strategy would be to include the GPL-2 license file (as the skeleton currently proposes). Otherwise we would need to include any newer version of the license, like e.g. GPL-5 gets released resulting in the need to re-build the package to include that license version, which imho does not make much sense.

tldr; the current skeleton proposal make sense and updating recipes to those also.

Reply:
@dbast Thanks for the clear and decisive answer! I agree your reasoning makes sense, and this is also my preference. I'll proceed with updating the "GPL (>=2)" recipes to bundle the GPL-2 license (instead of GPL-3).